reactioncommerce/reaction
# 2452 fix on UI icon toolbar

Resolves #2452 
**Testing**
- Start the app and Login as admin
- Try adjusting the page height and notice that the main `dashboard icon` and the `orders icon` do not overlap
